### PR TITLE
Fix "Prefer format() over string interpolation operator" issue.

### PR DESCRIPTION
**Pull request by ghost**
This pull request automatically fixes all occurrences of the following issue:

Issue type: [Prefer `format()` over string interpolation operator](https://www.quantifiedcode.com/app/issue_class/4ACGxFj1)
Issue details: [https://www.quantifiedcode.com/app/project/gh:bastiak:ipa-devel-tools?groups=code_patterns/%3A4ACGxFj1](https://www.quantifiedcode.com/app/project/gh:bastiak:ipa-devel-tools?groups=code_patterns/%3A4ACGxFj1)

To adjust the commit message or the actual code changes, just [rebase](https://git-scm.com/docs/git-rebase) or [cherry-pick](https://git-scm.com/docs/git-cherry-pick) the commits.
For questions or feedback reach out to cody@quantifiedcode.com.

Legal note: We won't claim any copyrights on the code changes.

Cheers,
[Cody - Your code quality bot](https://www.quantifiedcode.com/cody)

---

This change is [Reviewable](https://reviewable.io/reviews/bastiak/ipa-devel-tools/10)

test

### CODE REVIEW

File: ipavagrant/ipaci.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def create(self):
             raise RuntimeError("Timeout during generating SSH keys")
         else:
             if proc.returncode is not None and proc.returncode != 0:
-                raise RuntimeError("Failed to generate SSH key: %s" % errs)
+                raise RuntimeError("Failed to generate SSH key: {0!s}".format(errs))
```
Comment (bastiak) [inline ipavagrant/ipaci.py]: __[flake8]__
* [E501] line too long (84 > 79 characters)

**Review by MartinBasti — CHANGES_REQUESTED** [review]
Test

**Review by MartinBasti — APPROVED** [review]

**Review by MartinBasti — CHANGES_REQUESTED** [review]
Test

File: ipavagrant/ipaci.py (hunk 1 above)
Comment (MartinBasti) [inline ipavagrant/ipaci.py:72]: LGTM

File: ipavagrant/vagrant.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def __init__(self, topology_path, config, num_replicas, num_clients):
         self.topology_path = topology_path
         self.config = config
 
-        self.network_octets = '192.168.%s' % random.randint(100, 200)
+        self.network_octets = '192.168.{0!s}'.format(random.randint(100, 200))
         self.ip_addrs = self._generate_ip_addresses(self.network_octets,
                                                     constants.IP_ADDR_FIRST)
```
Comment (MartinBasti) [inline ipavagrant/vagrant.py:72]: LGTM2

**Review by MartinBasti — APPROVED** [review]

**Review by MartinBasti — CHANGES_REQUESTED** [review]
test

**Comment by MartinBasti** [comment]
test

**Review by MartinBasti — APPROVED** [review]

**Comment by MartinBasti** [comment]
test

**Review by MartinBasti — CHANGES_REQUESTED** [review]
test

File: ipavagrant/ipaci.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Author: Martin Basti
 # See LICENSE file for license
 
-import sys
+importt sys
 import os
```
Comment (MartinBasti) [inline ipavagrant/ipaci.py:6]: This is bad, really

**Review by MartinBasti — CHANGES_REQUESTED** [review]

File: ipavagrant/ipaci.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # Author: Martin Basti
 # See LICENSE file for license
```
Comment (MartinBasti) [inline ipavagrant/ipaci.py:3]: test

Comment (MartinBasti) [reply ipavagrant/ipaci.py:3]: test2

Comment (MartinBasti) [reply ipavagrant/ipaci.py:3]: test3

**Comment by MartinBasti** [comment]
test

**Comment by MartinBasti** [comment]
test

File: ipavagrant/ipaci.py (hunk 4 above)
Comment (MartinBasti) [reply ipavagrant/ipaci.py:3]: test

File: ipavagrant/vagrant.py [hunk 5]
```diff
@@ -90,7 +90,7 @@ def _generate_ip_addresses(self, network_24, start_from):
         replicas = {}
         ip_addresses['replicas'] = replicas
         for k in range(1, self.num_replicas + 1):
-            replicas['replica%s' % k] = {
+            replicas['replica{0!s}'.format(k)] = {
```
Comment (MartinBasti) [inline ipavagrant/vagrant.py:93]: test

Comment (MartinBasti) [reply ipavagrant/vagrant.py:93]: cc

File: ipavagrant/vagrant.py [hunk 6]
```diff
@@ -99,7 +99,7 @@ def _generate_ip_addresses(self, network_24, start_from):
         clients = {}
         ip_addresses['clients'] = clients
         for k in range(1, self.num_clients + 1):
-            clients['client%s' % k] = {
+            clients['client{0!s}'.format(k)] = {
                 'ip': '{}.{}'.format(network_24, i),
```
Comment (MartinBasti) [inline ipavagrant/vagrant.py:103]: cccc

**Review by MartinBasti — CHANGES_REQUESTED** [review]

**Review by MartinBasti — APPROVED** [review]

**Comment by MartinBasti** [comment]
test

**Review by MartinBasti — CHANGES_REQUESTED** [review]
test

**Comment by MartinBasti** [comment]
test

**Pull request closed without merging**